robottelo/performance/candlepin.py

```diff
@@ -25,9 +24,9 @@ class Candlepin(object):
     """
 
     # parameters for Candlepin Authentication
-    username = conf.properties['foreman.admin.username']
-    password = conf.properties['foreman.admin.password']
-    serverhost = conf.properties['main.server.hostname']
+    username = settings.server.admin_username
```

Comment:
@Ichimonji10 you made a really good comment. I that particular case I dropped the class attr and used where necessary. Will adjust the remaining depending on the case.